x-pack/legacy/plugins/telemetry/public/services/telemetry_opt_in.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+export { TelemetryOptInProvider } from '../../../../../../src/legacy/core_plugins/kibana/public/home/telemetry_opt_in'; // eslint-disable-line
```

Comment:
`// eslint-disable-line` is not necessary here

This include needs to be tested out with build Kibana too, since in built Kibana, the location of `x-pack` moves. Relative imports out of x-pack this way, do not work afaik

Reply:
I'm disabling eslint as it spreads the `export` statement into 3 lines as the import string is long. Then the `@ts-ignore` does not ignore the right line. Moving the `@ts-ignore` line between those 3 lines of the `export` statement also makes the linter complain.

Reply:
I did test this in a production build and it works fine; there are places in x-pack where its requiring code from src as well in a 100 different files (searched x-pack folder for `../src` excluding test files)